App/Services/MutagenController.cs

```diff
@@ -534,25 +538,44 @@ private void StartDaemonProcess()
         Directory.CreateDirectory(_mutagenDataDirectory);
         var logPath = Path.Combine(_mutagenDataDirectory, "daemon.log");
         var logStream = new StreamWriter(logPath, true);
+        try
```

Comment:
Can you refactor this into a separate PR? It doesn't have anything to do with the remote picker.